content/rc/subscriptions/maintenance/_index.md

```diff
@@ -26,15 +26,32 @@ This includes, but is not limited to:
 - Adding more memory to a node
 - Applying security patches
 
-Redis will notify users by email when maintenance starts and ends. If Redis needs an action from a user to start maintenance, Redis will notify users with a reasonable amount of time before planned maintenance.
-
-For major upgrades or upgrades that might include breaking changes, users will receive an advance notification with sufficient time to prepare before the upgrade.
+Redis will notify users by email when maintenance starts and ends. For more details, see [Notifications](#notifications).
 
-If you want to receive notifications by email, make sure **Operational emails** are activated in your user settings in [Access Management]({{<relref "/rc/security/access-control/access-management">}}). 
+During maintenance, your application may disconnect from our service for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries.
```

Comment:
Besides my previous comment about adding the point about the latency, I suggest starting the section with "during a maintenance activity, your database will be operational, but...

Reply:
Fixed in [1bbee48](https://github.com/RedisLabs/redislabs-docs/pull/3194/commits/1bbee482e5a922619d09fe502aaa46a4107c6170).